winston/Tabber.swift

```diff
@@ -69,7 +69,13 @@ struct Tabber: View {
         .tabItem {
           VStack {
             Image(systemName: "person.fill")
-            Text("Me")
+              if let me = redditAPI.me {
+                  if let data = me.data {
+                      Text(showUsernameInTabBar ? data.name : "Me")
+                  }
+              } else {
+                  Text("Me")
+              }
```

Comment:
```suggestion
              if showUsernameInTabBar, let me = redditAPI.me, let data = me.data {
                  Text(data.name)
              } else {
                  Text("Me")
              }
```

Otherwise you'll have a failing case if the data can't be parsed